sample-app-cicd/gitops-workshop-redhat-registry-pull-secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
Created these secrets to see if I could get image pulling and pushing to work with a custom defined ServiceAccount instead of pushing auth data into the pipeline SA. Haven't got it working yet, but the secrets SHOULD contain the necessary auth info. Issue must be in further SA user config.